handler_wrapper.go

```diff
@@ -149,11 +160,13 @@ func (hw *HandlerWrapper) Metric(name string, value interface{}) {
 
 	s := coerceString(value)
 	if s != nil {
+		hw.Label("@iopipe/metrics")
```

Comment:
Does this get reported if a custom metric is generated that starts with `@iopipe/` or when the user specifically defines their own custom metric? If this adds the metric regardless of its name then it would signal that metrics exists despite them purely being automatic metrics (like from event info).

Reply:
Will add this check.